src/factorizations/svd.jl

```diff
@@ -46,21 +54,19 @@ function MatrixAlgebraKit.initialize_output(
     row, col = Int.(Tuple(bI))
     nrows = brows[row]
     ncols = bcols[col]
-    slengths[col] = min(nrows, ncols)
+    s_axeses[col] = min(nrows, ncols)
```

Comment:
How does this work with the graded information? It seems like this is very integer specific.

Reply:
I've made the code clearer with better names and by using `argmin`.